src/descriptor/mod.rs

```diff
@@ -1655,7 +1654,7 @@ mod tests {
         .unwrap();
         assert_eq!(
             *descriptor.script_code().unwrap().as_bytes(),
-            Vec::<u8>::from_hex("76a91479091972186c449eb1ded22b78e40d009bdf008988ac").unwrap()[..]
+            hex::decode_to_vec("76a91479091972186c449eb1ded22b78e40d009bdf008988ac").unwrap()[..]
```

Comment:
Out of interest did you find yourself cursing the f***ing `hex-conservative` devs or were you happy enough during the upgrade?

Reply:
I was happy enough, but the lack of encoding support is idiotic.

Reply:
True that.